docs/appendices/data-persistence.md

```diff
@@ -2,9 +2,9 @@
 
 **Please note this is a simplified explanation of how Docker stores data in relation to a typical ISLE installation.** This document assumes the use of the docker-compose command and docker-compose.yml, though there are other ways to specify how docker uses volumes and bind mounts.  For more complete descriptions of bind mounts and volumes, and their management, please see the official docker documentation.
 
-**Data**, in this context, can include digital objects, files, logs, code, or information stored in mysql, solr, or fedora.  Essentially, anything typically written to or read from disk in some format, is data. 
+**Data**, in this context, can include digital objects, files, logs, code, or information stored in MySQL, SLR, or fedora.  Essentially, anything typically written to or read from disk in some format, is data. 
```

Comment:
Should be Solr not SLR?  And Fedora should be capitalized as well.

Reply:
Oops, I must have typed "2" instead of "1" when using Aspell. The first suggestion (#1) is usually the correct one for when I want to capitalize a community word, like Solr, that I have added to the Aspell word list. (Just giving some background of how my process goes.)

I will fix this. 

Also, need to do another pass to check all the "Fedora" entries, since I needed to double check if it how it should be capitalized. (I checked today and their site says "Fedora".)